### DISCUSSION TITLE
[Bug] Feeds for Lodestone not being read correctly

### DISCUSSION THREAD

**Discussion by dlmw** [FreshRSS/FreshRSS#6596] [Q&A] [answered]
### Describe the bug

I added the feeds for Lodestone (Final Fantasy XIV) but it doesn't display them.

### To Reproduce

1. Add "https://eu.finalfantasyxiv.com/lodestone/news/topics.xml" or "https://eu.finalfantasyxiv.com/lodestone/news/news.xml" as a feed
2. Go back to the main page and click on the feed that was added
3. Notice "There are no articles to show."

### Expected behavior

Articles should be displayed

### FreshRSS version

1.24.0

### Environment information

- Database version: Postgres 16.3
- PHP version: 8.2.19
- Installation type: Docker
- Web server type: Unsure
- Device: PC
- OS: openSUSE Tumbleweed
- Browser: LibreWolf 127.0.1-1

**Comment by Alkarex** [comment, marked as answer]
Try to change the user-agent to `curl`:

![image](https://github.com/FreshRSS/FreshRSS/assets/1008324/2d5ca205-7892-4bfd-aa9a-d30da4594a0c)

**Comment by dlmw** [comment]
Thanks that fixed it! What was the issue exactly?

**Reply by Alkarex** [reply]
The server seems to be filtering on user-agent, which is a relatively common bad practice, unfortunately
